presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -538,6 +538,6 @@ private void toString(StringBuilder builder, int indent)
 
     private static String indentString(int indent)
     {
-        return Strings.repeat("    ", indent);
+        return Strings.repeat("    ", indent) + "- ";
```

Comment:
Seems weird for the dash to be part of the indent string. Move it to whoever calls it?